src/main/kotlin/com/criptext/mail/scenes/composer/ComposerScene.kt

```diff
@@ -223,6 +226,14 @@ interface ComposerScene {
                     R.drawable.arrow_down).into(imageViewArrow)
         }
 
+        override fun showStartGuideAttachments(hostActivity: IHostActivity) {
```

Comment:
Don't pass the IHostActivity, use the observer to let the Controller know want you want him to do with the hostactivity.